barretenberg/cpp/src/barretenberg/stdlib/primitives/field/field.cpp

```diff
@@ -25,8 +25,8 @@ field_t<Builder>::field_t(Builder* parent_context)
 template <typename Builder>
 field_t<Builder>::field_t(const witness_t<Builder>& value)
     : context(value.context)
-    , additive_constant(0)
-    , multiplicative_constant(1)
+    , additive_constant(bb::fr::zero())
```

Comment:
I thought these were deprecated: https://github.com/AztecProtocol/aztec-packages/issues/1270 . I like (and prefer) them, but would be good to be consistent and/or close the issue :)

Reply:
> I thought these were deprecated: #1270 . I like (and prefer) them, but would be good to be consistent and/or close the issue :)

Curious why you prefer them? You think it makes the intent more explicit or something?

Reply:
Ah, no strong preference at all in any case! I guess when I first saw it I though "ah, sure, this will be precomputed once and then just used, nice" but was not aware many things in `fr` are constexpr. Taking constexpr into account, I actually wonder if `0`/`1` is better, or if `::zero()`/`::one()` can be used in constexprs (probably not if they are underlying `static` vars, I haven't checked).

In that case, using 0/1 everywhere might make more sense. But again, no strong preference really. If there were no constexpr constructors then I would indeed maybe have a slight inclination towards `::zero()`/`::one()`.